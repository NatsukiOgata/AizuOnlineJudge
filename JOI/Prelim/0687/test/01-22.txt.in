90
1407 1248 1042 1411 1410 1214 1308 1224 1183 1057 1034 1292 1300 1211 1296 1268 1051 1173 1409 1150 1294 1279 1396 1016 1476 1446 1148 1395 1112 1200 1101 1331 1270 1165 1318 1434 1188 1157 1333 1391 1029 1129 1229 1025 1177 1130 1071 1005 1403 1458 1500 1171 1199 1298 1394 1231 1217 1213 1356 1417 1488 1310 1406 1326 1453 1092 1418 1357 1120 1351 1091 1168 1166 1243 1336 1153 1027 1334 1457 1122 1094 1000 1108 1232 1023 1431 1451 1201 1290 1372
