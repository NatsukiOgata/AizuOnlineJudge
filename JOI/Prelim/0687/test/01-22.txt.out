61794
49116
